homeassistant/components/wled/light.py

```diff
@@ -312,7 +313,10 @@ async def async_turn_off(self, **kwargs: Any) -> None:
     @wled_exception_handler
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Turn on the light."""
-        data = {ATTR_ON: True, ATTR_SEGMENT_ID: self._segment}
+        data: dict[str, Any] = {
```

Comment:
I think this is a good place for `TypedDict`. There are many keys which are worth to be checked.

Reply:
I  think it's cleaner to just call the function with the arguments, instead  of having everything into a dict (typed or  not). However, either way, I think that's more suited to a future PR given the uncertainty in this one already.

Reply:
Yes, normal arguments are cleaner for sure.